migrations/1651305296248-createLogTable.ts

```diff
@@ -6,12 +6,16 @@ export class createLogTable1651305296248 implements MigrationInterface {
       `
             CREATE TABLE IF NOT EXISTS public.log
               (
-                  id integer NOT NULL,
-                  "serviceName" character varying COLLATE pg_catalog."default" NOT NULL,
+                  id SERIAL NOT NULL,
+                  "statusCode" integer,
+                  url character varying COLLATE pg_catalog."default" NOT NULL,
+                  scope character varying COLLATE pg_catalog."default",
                   status character varying COLLATE pg_catalog."default" NOT NULL,
-                  error character varying COLLATE pg_catalog."default" NOT NULL,
+                  method character varying COLLATE pg_catalog."default" NOT NULL,
+                  error character varying COLLATE pg_catalog."default" ,
                   "trackId" character varying COLLATE pg_catalog."default" NOT NULL,
-                  result character varying COLLATE pg_catalog."default" NOT NULL,
+                  result character varying COLLATE pg_catalog."default",
+                
```

Comment:
Please add updatedAt and CreatedAt Columns.
Please look into typeorm columns I think in this case this can be helpful: https://www.kindacode.com/snippet/typeorm-adding-created_at-and-updated_at-columns/#:~:text=In%20TypeORM%2C%20you%20can%20add,is%2C%20the%20current%20timestamp).

Reply:
CreateDateColumn, and UpdateDateColumn for automatically updating those columns without having to manually set them all the time.

But up to you if you want to set them manually in the methods.